symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/OboUserService.java

```diff
@@ -96,4 +96,24 @@
    * @see <a href="https://developers.symphony.com/restapi/reference#search-users">Search Users</a>
    */
   List<UserV2> searchUserBySearchQuery(@Nonnull UserSearchQuery query, @Nullable Boolean local);
+
+  /**
+   * Make a list of users to start following a specific user.
+   * {@link UserService#followUser(Long, List)}
+   *
+   * @param uid         The id of the user to be followed.
+   * @param followerIds List of ids of the followers.
+   * @see <a href="https://developers.symphony.com/restapi/v20.9/reference#follow-user">Follow User</a>
+   */
+  void followUser(@Nonnull Long uid, @Nonnull List<Long> followerIds);
+
+  /**
+   * Make a list of users to stop following a specific user.
+   * {@link UserService#unfollowUser(Long, List)}
+   *
+   * @param uid         The id of the user to be unfollowed.
+   * @param followerIds List of the ids of the followers.
+   * @see <a href=https://developers.symphony.com/restapi/v20.9/reference#unfollow-user">Unfollow User</a>
```

Comment:
You missed the first double quote in `href`

Reply:
Eagle eyes! 🦅